### PR TITLE
Fix a couple of rule violations for the `mime-types` example scan

### PR DESCRIPTION
**Pull request by fviernau**
Fix violations in 4 packages which cause a high amount of false positives, to reduce the amount of errors quite a bit.

### CODE REVIEW

File: package-configurations/NPM/_/spdx-correct/3.1.0/vcs.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  reason: "TEST_OF"
+- pattern: "README.md"
+  reason: "DOCUMENTATION_OF"
+license_finding_curations:
+- path: "index.js"
+  reason: "CODE"
+  concluded_license: "Apache-2.0"
+  comment: "The code contains listings of license identifiers which do not apply.\
+    \ The only license text is in the header of the file stating that it is licensed\
+    \ under 'Apache-2.0'."
```
Comment (MarcelBochtler) [inline package-configurations/NPM/_/spdx-correct/3.1.0/vcs.yml:20]: Should we stick to multi line comments using`|` for consistency?

Comment (sschuberth) [reply package-configurations/NPM/_/spdx-correct/3.1.0/vcs.yml:20]: I tend to agree (not only for consistency, but also as it makes the comment more readable), but to be fair we should probably not insist on it until we have checks in place to enforce this, if we agree on the formatting.

Comment (MarcelBochtler) [reply package-configurations/NPM/_/spdx-correct/3.1.0/vcs.yml:20]: I don't insist on using multi line comments, that's why I just asked the question.

Comment (fviernau) [reply package-configurations/NPM/_/spdx-correct/3.1.0/vcs.yml:20]: Ok, then let's leave the formatting as-is for now. Note: I've formatted it with the `helper-cli` `package config` commands, `format` or `sort` respectively.

**Review by MarcelBochtler — APPROVED** [review]

File: package-configurations/NPM/_/spdx-correct/3.1.0/vcs.yml (hunk 1 above)
Comment (sschuberth) [reply package-configurations/NPM/_/spdx-correct/3.1.0/vcs.yml:20]: > I've formatted it with the `helper-cli` `package config` commands, `format` or `sort` respectively.

So, would you be fine with changing these commands to use `|` as it requires less quoting / escaping?

Comment (fviernau) [reply package-configurations/NPM/_/spdx-correct/3.1.0/vcs.yml:20]: > So, would you be fine with changing these commands to use `|` as it requires less quoting / escaping?

I'd be fine with that.

**Pull request merged**